tdrs-backend/tdpservice/reports/test/test_serializers.py

```diff
@@ -0,0 +1,25 @@
+"""Test report serializers."""
+
+import pytest
+from ..serializers import ReportFileSerializer
+
+@pytest.mark.django_db
+def test_serializer_with_valid_date(report):
+    """If a serializer has valid data it will return a valid object."""
```

Comment:
I like it but this only tests the valid "happy path". Without a test for the invalid data path to make sure the proper error messages are raised, etc, we're not fully testing validation.

Reply:
I agree, I'll set up a few more tests.

Reply:
I've added a case that checks for there being an `ImmutabilityError` when a report is updated.

Reply:
Sounds like a great strategy!

Reply:
Looks like the tests are red on CircleCI though?

Reply:
I thought they were passing when I stopped last night, and they seemed to be passing on the raft pr.

Reply:
Alright, glad to see the immutability test added below! and CircleCI now green.

One more note: 

The name of this test is `test_serializer_with_valid_date`, I believe it should be `test_serializer_with_valid_data`.

Reply:
@alexsoble fixed that

Reply:
Hmmm, maybe you updated it on the Raft repo but not here? I'm not seeing the change here